servicecontrol/plugins.md

```diff
@@ -112,11 +116,19 @@ The result of a custom check is either success or a failure (with a detailed des
 
 - [How to Develop Custom Checks for ServicePulse](/servicepulse/how-to-develop-custom-checks.md)
 
+#### Should I deploy the CustomChecks plugin to Production?
+Please ensure that you **do deploy** this plugin to your endpoint in production, as it sends relevant information that the developers have specifically coded to highlight as problems to the Ops department to take notice of and act when something fails. This information is displayed in the ServicePulse dashboard.
```

Comment:
Same here, perhaps just bake "this plugin allows you to write custom checks making sure that your production system is running properly...." to make it obvious that it's meant to be used in prod?